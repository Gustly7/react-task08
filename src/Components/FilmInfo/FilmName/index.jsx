import React, { Component } from 'react'

class FilmName extends Component {
  render () {
    return (
      <div className='FilmName'>
        {this.props.filmName}
      </div>
    )
  }
}

export default FilmName
