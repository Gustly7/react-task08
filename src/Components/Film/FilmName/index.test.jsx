import React, { Component } from 'react'
import FilmName from './index'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

describe('FilmName component test suite', () => {
  test('FilmName snapshot renders correctly', () => {
    const tree = renderer.create(
      <FilmName />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('FilmName div should have "FilmName" class', () => {
    const wrapper = shallow(
      <FilmName />
    )
    const elem = wrapper.find('div.FilmName')
    expect(elem.length).toBe(1)
  })
})
