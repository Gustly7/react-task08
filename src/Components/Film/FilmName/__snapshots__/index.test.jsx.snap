// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmName component test suite FilmName snapshot renders correctly 1`] = `
<div
  className="FilmName"
/>
`;
